.github/workflows/benchmarks.yaml

```diff
@@ -0,0 +1,63 @@
+name: benchmarks
+
+on:
```

Comment:
Instead of doing this on every push & PR, can we have this be invoked in a special case?

I think the easiest is to have a special string that we comment on PRs, like `/workflows/benchmarks`. Then in this workflow, we do:
```yaml
name: benchmarks

on:
  issue_comment:
    types: [created, edited]
    
jobs:
  run:
    runs-on: ubuntu-latest
    if: |
      github.event.issue.pull_request != null &&
      contains(github.event.comment.body, '/workflows/benchmarks')
```

We should then create a comment on the PR pointing to the workflow that was created:
```yaml
      - name: Running Benchmarks...
        uses: actions/github-script@0.8.0
        with:
          github-token: ${{secrets.GITHUB_TOKEN}}
          script: |
            github.issues.createComment({
              issue_number: context.issue.number,
              owner: context.repo.owner,
              repo: context.repo.repo,
              body: '[Running Benchmarks](https://github.com/${{github.repository}}/actions/runs/${{github.run_id}})'
            })
```

Reply:
awesome! let's test this :smile: